ChartsDemo-macOS/PlaygroundChart.playground/Pages/BarChart.xcplaygroundpage/Contents.swift

```diff
@@ -17,14 +17,12 @@
  */
 
 //: # Bar Chart
-//#-hidden-code
```

Comment:
Why?

Reply:
I have not been working, at least, not been working on my playground since I upgraded to Xcode 10